lib/taskana-core-test/src/test/java/acceptance/task/create/CreateTaskAccTest.java

```diff
@@ -0,0 +1,967 @@
+        .hasMessage("Classification of Attachment must not be null.");
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_ThrowException_When_CreatingTaskWithAttachmentObjectReferenceNull() {
+    Task task = createDefaultTask();
+    task.addAttachment(
+        TaskAttachmentBuilder.newAttachment()
+            .classificationSummary(defaultClassificationSummary)
+            .objectReference(null)
+            .channel("E-MAIL")
+            .received(Instant.parse("2018-01-15T00:00:00Z"))
+            .customAttributes(createSimpleCustomPropertyMap(3))
+            .build());
+
+    assertThatThrownBy(() -> taskService.createTask(task))
+        .isInstanceOf(InvalidArgumentException.class)
+        .hasMessageContaining("ObjectReference of Attachment must not be null.");
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_ThrowException_When_CreatingTaskWithNotExistingAttachmentClassification() {
+    ClassificationSummary classificationSummary =
+        classificationService
+            .newClassification("NOT_EXISTING", defaultClassificationSummary.getDomain(), "")
+            .asSummary();
+    Task task = createDefaultTask();
+
+    task.addAttachment(
+        TaskAttachmentBuilder.newAttachment()
+            .classificationSummary(classificationSummary)
+            .objectReference(defaultObjectReference)
+            .channel("E-MAIL")
+            .received(Instant.parse("2018-01-15T00:00:00Z"))
+            .customAttributes(createSimpleCustomPropertyMap(3))
+            .build());
+
+    ThrowingCallable call = () -> taskService.createTask(task);
+    ClassificationNotFoundException e =
+        catchThrowableOfType(call, ClassificationNotFoundException.class);
+    assertThat(e.getClassificationKey()).isEqualTo("NOT_EXISTING");
+    assertThat(e.getDomain()).isEqualTo(defaultClassificationSummary.getDomain());
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_ThrowException_When_CreatingTaskWithMissingAttachmentClassificationKey() {
+    Task task = createDefaultTask();
+    ClassificationSummary classificationSummary = new ClassificationSummaryImpl();
+    task.addAttachment(
+        TaskAttachmentBuilder.newAttachment()
+            .classificationSummary(classificationSummary)
+            .objectReference(defaultObjectReference)
+            .channel("E-MAIL")
+            .received(Instant.parse("2018-01-15T00:00:00Z"))
+            .customAttributes(createSimpleCustomPropertyMap(3))
+            .build());
+
+    assertThatThrownBy(() -> taskService.createTask(task))
+        .isInstanceOf(InvalidArgumentException.class)
+        .hasMessageContaining("ClassificationKey of Attachment must not be empty.");
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_FetchAttachmentClassification_When_CreatingTaskWithAttachments() throws Exception {
+    Task task = createDefaultTask();
+    ClassificationSummary classificationSummary =
```

Comment:
Why do you need this classificationSummary? You don't use it except in the assertions